drivers/console/Kconfig

```diff
@@ -87,6 +87,14 @@ config UART_CONSOLE_INPUT_EXPIRED_TIMEOUT
 	  Fixed amount of time which unit is milliseconds to keep the UART
 	  console in use flag true.
 
+config JAILHOUSE_DEBUG_CONSOLE
+	bool "Use JAILHOUSE_DEBUG console"
+	select CONSOLE_HAS_DRIVER
+	help
+	  Emit console messages to a jailhouse hypervisor debug console.
+	  Useful in board bring-up if there aren't any working serial
+	  drivers.
```

Comment:
Although the "jailhouse console" is not an ARM-specific construct, this config should depend on `ARM64` for now since the associated driver makes explicit use of A64 HVC instruction and does not support any other architectures (e.g. x86).

Reply:
fix in next version.